src/crypto/algorithms/megolm.js

```diff
@@ -656,6 +665,7 @@ MegolmDecryption.prototype.onRoomKeyEvent = function(event) {
     this._olmDevice.addInboundGroupSession(
         content.room_id, senderKey, sessionId,
         content.session_key, event.getKeysClaimed(),
+        exportFormat,
```

Comment:
Will the `getKeysClaimed` be correct here?
I suspect it should be claiming the keys of the original sender here?